gufe/proteincomponent.py

```diff
@@ -39,6 +52,9 @@ def from_rdkit(cls, rdkit: RDKitMol, name=""):
     def to_rdkit(self) -> RDKitMol:
         return Chem.Mol(self._rdkit)
 
+    def to_openmm_PDBFile(self):
+        return self._openmm_rep
```

Comment:
```suggestion
        if HAS_OPENMM:
            return self._openmm_rep
        else:
            raise ImportError("OpenMM module is not available")
```

Possible suggestion here, I feel like returning `None` could be dangerous / misleading. You might as well break early if someone wants to do something they can't do?